src/svm/transactionUtils.ts

```diff
@@ -51,13 +52,14 @@ export async function sendTransactionWithLookupTable(
       addresses: lookupAddresses.slice(i, i + maxExtendedAccounts),
     });
 
-    await web3.sendAndConfirmTransaction(connection, new web3.Transaction().add(extendInstruction), [sender], {
-      skipPreflight: true, // Avoids recent slot mismatch in simulation.
-    });
+    await web3.sendAndConfirmTransaction(connection, new web3.Transaction().add(extendInstruction), [sender]);
```

Comment:
shall we also set this to "confirmed", otherwise might default to "max"

Reply:
sure & done.